lib/auth/authclient/http_client.go

```diff
@@ -424,6 +424,8 @@ func (c *HTTPClient) UpsertAuthServer(ctx context.Context, s types.Server) error
 }
 
 // GetAuthServers returns the list of auth servers registered in the cluster.
+//
+// Deprecated: Prefer paginated variant [APIClient.ListAuthServers].
```

Comment:
Let's move both of these deprecated methods to [lib/auth/authclient/httpfallback](https://github.com/gravitational/teleport/blob/master/lib/auth/authclient/httpfallback.go).